.github/workflows/reusable-code-quality.yml

```diff
@@ -70,6 +70,29 @@ jobs:
         if: steps.check_linter_file.outputs.files_exists == 'true'
         run: vendor/bin/parallel-lint -j 10 . --show-deprecated --exclude vendor --exclude .git --checkstyle | cs2pr
 
+  lint-gherkin: #----------------------------------------------------------------
+    name: Lint Gherkin Feature files
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out source code
+        uses: actions/checkout@v4
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+
+      - name: Check existence of .gherkin-lintrc file
+        id: check_gherkin_lintrc_file
+        uses: andstor/file-existence-action@v3
+        with:
+          files: ".gherkin-lintrc"
+
+      - name: Download lint rules
+        if: steps.check_gherkin_lintrc_file.outputs.files_exists == 'false'
+        run: curl https://raw.githubusercontent.com/wp-cli/.github/refs/heads/add-gherkin-linter/.gherkin-lintrc -o $RUNNER_TEMP/.gherkin-lintrc
+
+      - name: Run linter
+        run: npx --yes gherkin-lint -c $RUNNER_TEMP/.gherkin-lintrc
```

Comment:
Ah, of course this only works if the file had to be downloaded there 🤦  My bad.

So we need to either conditionally run with `-c $RUNNER_TEMP/.gherkin-lintrc` and without it (if the file already exists), or just always copy the file to the temp directory 🤷

Reply:
Yes. Therefore I removed file exists check.